packages/web/src/scss/components/Navigation/_NavigationLink.scss

```diff
@@ -0,0 +1,52 @@
+@use '@tokens' as tokens;
+@use '../../tools/typography';
+@use 'theme';
+
+.NavigationLink {
```

Comment:
There is a "hanging" underline on click and release outside:
![image](https://github.com/user-attachments/assets/c6c0d5e9-8446-4b29-a1fb-6d7cc397667b)

Reply:
solved. Forgot to remove underline in the active state.